### DISCUSSION TITLE
When will play 3.0.0 be released ?

### DISCUSSION THREAD

**Discussion by asksagarjain** [playframework/playframework#12180] [Q&A] [answered]
Do we have the date for the release of 3.0.0
It’s not present on maven central

**Comment by mkurz** [comment]
It is on maven central, but under `org.playframework`: https://repo1.maven.org/maven2/org/playframework/
See 
* https://www.playframework.com/documentation/3.0.x/Highlights30
* https://www.playframework.com/documentation/3.0.x/Migration30

I am just preparing last steps for the announcement.

**Comment by mkurz** [comment, marked as answer]
Play 2.9 and 3.0 have been released now: https://twitter.com/playframework/status/1721576472047911007
